browser/src/StatsPage/StatsPageTables/InferredSexPerGeneticAncestryTables.tsx

```diff
@@ -0,0 +1,374 @@
+import React from 'react'
+
+import { DownloadElementAsPNGButton } from '../DownloadFigure'
+
+import {
+  StatsTable,
+  StatsTableHeaderRow,
+  StatsTableSubHeaderRow,
+  StatsTableBody,
+  StatsTableCaption,
+  StatsTableFooter,
+} from './TableStyles'
+
+export const InferredSexAllV4Table = () => {
+  const elementId = 'inferred-sex-by-genetic-ancestry-group-table'
+
+  return (
+    <div>
+      <StatsTable>
+        <thead>
+          <StatsTableHeaderRow>
+            <th colSpan={2}>Genetic Ancestry</th>
+            <th colSpan={3}>v4 Exomes</th>
+            <th colSpan={3}>v4 Genomes</th>
+            <th colSpan={3}>Combined</th>
+          </StatsTableHeaderRow>
+          <StatsTableSubHeaderRow>
+            <th>Genetic Ancestry</th>
+            <th>Genetic Ancestry </th>
+            <th>Sample Count</th>
+            <th>XX</th>
+            <th>XY</th>
+            <th>Sample Count</th>
+            <th>XX</th>
+            <th>XY</th>
+            <th>Sample Count</th>
+            <th>XX</th>
+            <th>XY</th>
+          </StatsTableSubHeaderRow>
+        </thead>
+        <StatsTableBody>
+          <tr>
```

Comment:
Rather than have a long stretch of literal HTML in here I probably would have iterated over a collection of objects each representing a single table row's data, and rendered live from that. That said I don't think it's worth redoing it that way at this juncture, but it's something to keep in mind for next time.